{"name": "test_network_setup_create_network_invalid_org_id", "status": "passed", "description": "\n        Test network_setup create network failure (invalid org id)\n        ", "attachments": [{"name": "stdout", "source": "54092c46-4cae-4e9a-a63c-bcc770f0004f-attachment.txt", "type": "text/plain"}], "start": 1661841035446, "stop": 1661841035846, "uuid": "9db6e108-0679-49c8-b3e1-7047936ddd52", "historyId": "edf3c257a0b8535c34b2234d2e1ef50d", "testCaseId": "59bcff49fa49a5d2716e5aa95263ad72", "fullName": "tests.shasta_be.test_network.TestNetwork#test_network_setup_create_network_invalid_org_id", "labels": [{"name": "tag", "value": "regression"}, {"name": "parentSuite", "value": "tests.shasta_be"}, {"name": "suite", "value": "test_network"}, {"name": "subSuite", "value": "TestNetwork"}, {"name": "host", "value": "GURM-CGW1JG3"}, {"name": "thread", "value": "795-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.shasta_be.test_network"}]}